alf/config_helpers.py

```diff
@@ -124,8 +125,10 @@ def adjust_config_by_multi_process_divider(multi_process_divider: int = 1):
     For example, if we run 64 environments in parallel for single process
     settings, the value needs to be overriden with 16 if there are 4 identical
     processes running DDP training to achieve parity.
+
```

Comment:
Can you also list the configs which are adjusted in doc-string?

Reply:
Good point. Updated.